zerver/forms.py

```diff
@@ -206,19 +206,49 @@ def send_mail(self, subject_template_name, email_template_name,
                    from_name="Zulip Account Security",
                    from_address=FromAddress.NOREPLY, context=context)
 
+    def send_mail_to_non_existent_user(self, to_email):
+        # type: (str) -> None
+        """
+        This function is used to send an email when the email address does
+        not belong to any active Zulip account.
+        """
+        context = {'email': to_email}  # type: Dict[str, Text]
+        realm = get_realm(get_subdomain(getattr(self, 'request')))
+        org_name = u'Zulip'
```

Comment:
I would phrase these next few lines as and `if/else`, since it's only six lines of code, or put a `# until we know better` type comment.